home.html

```diff
@@ -41,31 +42,138 @@ <h1>
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/linkedin-brands.svg" alt="" class="icons" />
+                </a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/medium-brands.svg" alt="" class="icons"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/github-brands.svg" alt="" class="icons"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/angellist-brands.svg" alt="" class="icons"
+                /></a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="mt-3"></div>
+          <a href="#" class="btn btn-resume">Get my resume</a>
+        </section>
+        <!-- End about me section -->
+        <!-- Work experience section & work skills-->
+        <h5>Experience</h5>
+        <section class="experience">
+          <div class="expitem">
+            <div class="inneritem">
+              <img
+                src="assets/frontend-icon-4.png"
+                alt=""
+                width="60"
+                height="60"
+              />
+            </div>
+
+            <div class="inneritem">
+              <h3>Front end</h3>
+            </div>
+            <div class="inneritem">
+              <p>
+                I have experience in HTML5 and CSS and java script, therefore I
+                am able to build any kind of website using these technologies.
+              </p>
+            </div>
+          </div>
+
+          <div class="expitem">
+            <div class="inneritem">
+              <img
+                src="assets/work-backend.png"
+                alt=""
+                width="60"
+                height="60"
+              />
+            </div>
+            <div class="inneritem">
+              <h3>Back end</h3>
+            </div>
+            <div class="inneritem">
+              <p>
+                I have experience in Node js and PHP for building backend
+                projects, therefore I am able to build any kind of back project.
+              </p>
+            </div>
+          </div>
+          <div class="expitem">
+            <div class="inneritem">
+              <img
+                src="assets/work-crossplatform.png"
+                alt=""
+                width="60"
+                height="60"
+              />
+            </div>
+            <div class="inneritem">
+              <h3>Cross platform mobile</h3>
+            </div>
+            <div class="inneritem">
+              <p>
+                I have experience in Flutter , there for I am able to build any
+                kind of mobile application using flutter.
+              </p>
+            </div>
+          </div>
+        </section>
```

Comment:
- [ ] Following the original design is one of the project's requirements. This section looks totally different from [the original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Kindly make the required changes. Following the design is an important skill for a front-end developer 😉